Equilobe.DailyReport.Models/ReportPolicy/JiraGeneratedProperties.cs

```diff
@@ -42,7 +42,7 @@ public class JiraGeneratedProperties
         [XmlIgnore]
         public string ProjectManager { get; set; }
 
-        public string RootPath { get; set; }
+        public string RootPath { get { return Path.GetFullPath(ProjectName); } }
```

Comment:
@DanBadea 
modificarea asta e necesara la eliminarea altei metode, caz in care, doar actual functioneaza si pe partea de web
inainte se facea apel metoda SetRootPath care seta acest rootPath, deci de ce sa nu se faca aici si sa o fac explicit ?